specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```
@@ -2128,6 +2128,278 @@
+        "operationId": "Snapshot_Take",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "Request body for taking a snapshot.",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/TakeSnapshotRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The snapshot taking task was queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Operation location with an operation id used to track the progress of taking snapshot. The returned id is the operation id, rather than snapshot id. Snapshot id can be obtained only when the operation status becomes \"succeeded\" in OperationStatus - Get.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Take a snapshot example": {
+            "$ref": "./examples/TakeSnapshot.json"
+          }
+        }
+      },
+      "get": {
+        "description": "List all accessible snapshots with related information, including snapshots that were taken by the user, or snapshots to be applied to the user (subscription id was included in the ApplyScope in Snapshot - Take).",
```

I think the "ApplyScope" here should be "applyScope".
```suggestion
        "description": "List all accessible snapshots with related information, including snapshots that were taken by the user, or snapshots to be applied to the user (subscription id was included in the applyScope in Snapshot - Take).",
```

---

Done, thanks.